catalogue/webapp/components/Download/Download.js

```diff
@@ -140,100 +143,115 @@ const Download = ({
+                        [spacing({ s: 2 }, { margin: ['left'] })]: true,
+                      })}
+                    >
+                      JPG
+                    </span>
                   </span>
-                </span>
-              </a>
-            </li>
-          </ul>
-        </DownloadOptions>
-        <div className="flex-inline flex--v-center">
-          {iiifImageLocationLicenseId && (
-            <span
-              className={classNames({
-                'inline-block': true,
-                [spacing({ s: 2 }, { margin: ['right'] })]: true,
-              })}
-            >
-              <License subject={''} licenseType={iiifImageLocationLicenseId} />
-            </span>
+                </a>
+              </li>
+            </ul>
+          ) : (
+            <ul className="plain-list no-margin no-padding">
+              <li>Download option</li>
+              <li>Download option</li>
+              <li>Download option</li>
+            </ul>
           )}
-          {iiifImageLocationCredit && (
-            <span
-              className={classNames({
-                'inline-block': true,
-                [spacing({ s: 2 }, { margin: ['right'] })]: true,
-              })}
+        </DownloadOptions>
+        {!dummyVersion && (
+          <div className="flex-inline flex--v-center">
+            {iiifImageLocationLicenseId && (
+              <span
+                className={classNames({
+                  'inline-block': true,
+                  [spacing({ s: 2 }, { margin: ['right'] })]: true,
+                })}
+              >
+                <License
+                  subject={''}
+                  licenseType={iiifImageLocationLicenseId}
+                />
+              </span>
+            )}
+            {iiifImageLocationCredit && (
+              <span
+                className={classNames({
+                  'inline-block': true,
+                  [spacing({ s: 2 }, { margin: ['right'] })]: true,
+                })}
+              >
+                Credit: {iiifImageLocationCredit}{' '}
+              </span>
+            )}
+            <a
+              href="#licenseInformation"
+              className={font({ s: 'HNM5', m: 'HNM4' })}
             >
-              Credit: {iiifImageLocationCredit}{' '}
-            </span>
-          )}
-          <a
-            href="#licenseInformation"
-            className={font({ s: 'HNM5', m: 'HNM4' })}
-          >
-            <span className="flex-inline flex--v-center nowrap">
-              <Icon name="arrowSmall" extraClasses="icon--90" />
-              Can I use this?
-            </span>
-          </a>
-        </div>
+              <span className="flex-inline flex--v-center nowrap">
+                <Icon name="arrowSmall" extraClasses="icon--90" />
+                Can I use this?
+              </span>
+            </a>
+          </div>
+        )}
       </div>
     </div>
   );
```

Comment:
I'd feel more comfortable copy / pasting the component into another and including it in the parent component over refactoring the one in prod.